libc/CMakeLists.txt

```diff
@@ -843,9 +845,14 @@ target_link_libraries(oelibc
   PUBLIC oesyscall oecore oelibc_includes
   PRIVATE oelibasm)
 
+# Ensure the libc global initialization function is linked
+target_link_libraries(oelibc
+    INTERFACE
+    -Wl,-u,__oe_init_c)
```

Comment:
If a caller doesn't use cmake to build their enclave, they will likely not add this flag. Need to look for an in-source way to specify this. In theory the `used` attribute does this but for obvious reasons it is not used by the linker.